src/Includes/Helpers.php

```diff
@@ -113,7 +118,15 @@ public static function display_toggle_switch( $name ) {
 	 * @return array
 	 */
 	public static function get_settings() {
-		return get_option( 'plausible_analytics_settings', [] );
+		$settings = get_option( 'plausible_analytics_settings', [] );
+
+		// Keep around for backwards compatibility reasons.
+		$track_outbound_links = apply_filters( 'plausible_analytics_enable_outbound_links', isset( $settings['outbound-links'][0] ) ? $settings['outbound-links'][0] : true );
+		if ( $track_outbound_links ) {
+			$settings['outbound-links'][0] = 1;
+		}
+
+		return $settings;
```

Comment:
I'm not 100% happy with this bit of code, as it's... not ideally functional. Would love to hear other suggestions.